CMakeLists.txt

```diff
@@ -85,7 +85,6 @@ set(flags_to_test
     -Wextra
     -Wformat
     -Wformat-security
-    -fvisibility=hidden
```

Comment:
I removed this since it was causing issues with building so with functions being hidden